.coveragerc

```diff
@@ -1,3 +1,12 @@
 [run]
 include =
-  pycobertura/*
\ No newline at end of file
+  pycobertura/*
+omit=
+  **/__pycache__
+  **/dummy*
+  */templates
+  build
+  docs/_build
+  *.egg
+  .tox
+  *.venv
```

Comment:
Nice, that looks much more compact than what we had, thank you!

Can you help me understand one thing? If we have `include = pycobertura/*`, then why do we need to omit `build/`, `docs/`, or anything else outside the `pycobertura/` directory? Shouldn't they get omitted automatically?

I just pulled your branch and ran the tests with and without the "omit" parameter and I'm getting the same coverage output:

```
py38 run-test: commands[2] | coverage report -m
Name                                Stmts   Miss  Cover   Missing
-----------------------------------------------------------------
pycobertura/__init__.py                 2      0   100%
pycobertura/cli.py                     78      0   100%
pycobertura/cobertura.py              206      0   100%
pycobertura/filesystem.py              87      0   100%
pycobertura/reporters.py              192      0   100%
pycobertura/templates/__init__.py       0      0   100%
pycobertura/templates/filters.py       12      0   100%
pycobertura/utils.py                  118      1    99%   38
-----------------------------------------------------------------
TOTAL                                 695      1    99%
py38 run-test: commands[3] | coverage xml
Wrote XML report to coverage.xml
__________________________________________________________________________________________ summary ___________________________________________________________________________________________
  py38: commands succeeded
  congratulations :)
```

Am I missing something else?

Also, I can confirm that the filename attributes in the XML are correctly prefixed!

```
cat coverage.xml | grep filename | awk '{print $3}'
filename="pycobertura/__init__.py"
filename="pycobertura/cli.py"
filename="pycobertura/cobertura.py"
filename="pycobertura/filesystem.py"
filename="pycobertura/reporters.py"
filename="pycobertura/utils.py"
filename="pycobertura/templates/__init__.py"
filename="pycobertura/templates/filters.py"
```

🎉🎉🎉🎉🎉🎉

Reply:
Thanks, and yes I can remove this with 7446985b5da31bf40b5f6e0e03c4c7bcd829321e. When I was testing locally, I did not always have `pytest.ini` and then the `omit` part in `.coveragerc` will be necessary.